examples/merge_tag.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# Merge multiple tag/value file into one.
+# Usage: merge_tag.py <tagvaluefile1> <tagvaluefile2> ...
+
+if __name__ == '__main__':
+    import sys
```

Comment:
Can you make this whole code block as a proper function?
And sort and move imports at the top?